addons/mail/models/ir_actions_server.py

```diff
@@ -39,19 +39,20 @@ class ServerActions(models.Model):
         'Subscribe Recipients', compute='_compute_mail_post_autofollow',
         readonly=False, store=True)
     mail_post_method = fields.Selection(
-        selection=[('email', 'Email'), ('comment', 'Post as Message'), ('note', 'Post as Note')],
-        string='Send as',
+        selection=[('email', 'Email'), ('comment', 'Email and Message'), ('note', 'Email and Note')],
```

Comment:
Note that this is untrue as it does a message_post hence respect preferences of people, like those using Inbox. That is why we used the post / log notation like in discuss, and not the "email" as it does not really send emails, it posts messages that maybe sends notifications.

Reply:
@tde-banana-odoo I had a heck of a bad time trying to understand what did what here (most notably for SMS which is very confusing)
so to recap:
- 'email' send the email via the template (actual email), bypasses mail.thread fully
- 'comment' posts a message with the template on the record. nothings less nothing more (so output depends on user config)
- 'note' does the same with the internal note subtype

correct?

Reply:
@bouvyd Yep :D note that we could change a bit and say: "post to followers" (message_post) or "send mailing" (simulates a composer in mailing mode). And remove the template-based email sending. But that's not the purpose of this task I suppose.